pkg/pillar/types/volumetypes.go

```diff
@@ -142,6 +142,11 @@ func (status VolumeStatus) PathName() string {
 		status.GenerationCounter, strings.ToLower(status.ContentFormat.String()))
 }
 
+// ZVolName returns name of zvol for volume
+func (status VolumeStatus) ZVolName() string {
+	return fmt.Sprintf("%s/%s.%d", VolumeZFSPool, status.VolumeID.String(), status.GenerationCounter)
```

Comment:
I suspect me might need a vault and a clear pathname for zfs just like for ext*, so having a constent VolumeZFSPool might be suboptimal.